src/components/literacy-material/literacy-material.service.ts

```diff
@@ -291,21 +239,26 @@ export class LiteracyMaterialService {
       );
     }
 
-    const secured = parseSecuredProperties(result.propList, result.permList, {
-      name: true,
-    });
-
-    const scriptureReferences = await this.listScriptureReferences(
-      literacyMaterialId,
+    const scriptureReferences = await this.scriptureRefService.list(
+      id,
+      'LiteracyMaterial',
```

Comment:
Is this label really needed? Can't we just use the id with the 'BaseNode' label? Do we not index on `id` properties?

Reply:
Yeah. right.

Reply:
Can use `BaseNode`.